pages/linux/btrbk.md

```diff
@@ -0,0 +1,24 @@
+# btrbk
+
+> A tool for creating snapshots and remote backups of btrfs subvolumes.
+> More information: <https://digint.ch/btrbk/doc/readme.html>.
+
+- Print statistics about configured subvolumes and snapshots:
+
+`sudo btrbk stats`
+
+- List configured subvolumes and snapshots:
+
+`sudo btrbk list`
+
+- Print what run command would do:
+
+`sudo btrbk --verbose dryrun`
+
+- Run backup routines verbosely, show progress bar:
+
+`sudo btrbk --progress --verbose run`
+
+- Only create snapshots for configured subvolumes:
```

Comment:
```suggestion
- Only create snapshots for configured sub-volumes:
```

Reply:
BTRFS tooling uses subvolumes in `btrfs subvolume` sub-command, [docs](https://btrfs.readthedocs.io/en/latest/Subvolumes.html) and [manpages](https://manpages.org/btrfs-subvolume/8). I'd rather stay consistent.